README.md

```diff
@@ -56,14 +56,38 @@ expect(response.body).to match_snapshot("api/resources_index")
 
 # Using plain text instead of parsing JSON
 expect(response.body).to match_snapshot("api/resources_index", format: :text)
+```
+
+#### Matcher options
+
+##### dynamic_attributes
 
+Using `dynamic_attributes` inline allows to ignore attributes for a specific snapshot.
+This is useful to ignore changing attributes, like `id` or `created_at`.
+Notice that **all** nodes matching those (nested or not) will be ignored. Usage:
```

Comment:
In my scenario, the key is `in`:
`"time_zone" => {"presence"=>[{"message"=>"can't be blank"}], "inclusion"=>[{"in"=>[1,2,3]}]}`
The other two instances of `in` on the same test file will also be ignored.

I'm wondering on the long term impact.  If we later add a way to be more specific, it would be a breaking change.
🤔

Reply:
True, the common case is to ignore a unique attribute on a resource.